content/2022-munich/schedule.md

```diff
@@ -0,0 +1,332 @@
+---
+title: Schedule
+---
+
+## PromCon 2022 Schedule
+
+<table class="table schedule-table">
+  <tr class="day">
+    <th colspan="3">Day 1 - Tuesday, November 8</th>
+  </tr>
+  <tr class="break">
+    <td>8:15</td>
+    <td>Breakfast and Registration</td>
+    <td></td>
+  </tr>
+   <tr class="talk">
+    <td>9:00</td>
+    <td>
+      <a href="talks/">Welcome</a>
+    </td>
+    <td>
+      <a href="speakers/richard-richih-hartmann.md">Richard RichiH Hartmann</a>
+      <br>
+      <b>Grafana Labs</b>
+    </td>
+  </tr>
+   <tr class="talk">
+    <td>9:15</td>
+    <td>
+      <a href="talks/whats-new-in-prometheus-and-its-.md">What's new in Prometheus and its ecosystem</a>
+    </td>
+    <td>
+      <a href="speakers/julien.md">Julien</a>
+      <br>
+      <b>O11y</b>
+    </td>
+  </tr>
+   <tr class="talk">
+    <td>9:45</td>
+    <td>
+      <a href="talks/native-histograms-in-prometheus-.md">Native Histograms in Prometheus v2.41.0</a>
+    </td>
+    <td>
+      <a href="speakers/ganesh-vernekar.md">Ganesh Vernekar</a>
+      <br>
+      <b>Grafana Labs</b>
+    </td>
+  </tr>
+   <tr class="talk">
+    <td>10:15</td>
+    <td>
+      <a href="talks/promql-for-native-histograms.md">PromQL for Native Histograms</a>
+    </td>
+    <td>
+      <a href="speakers/björn-rabenstein.md">Björn Rabenstein</a>
+      <br>
+      <b>Grafana Labs</b>
+    </td>
+  </tr>
+  <tr class="break">
+    <td>10:45</td>
+    <td>Break</td>
+    <td></td>
+  </tr>
+   <tr class="talk">
+    <td>11:15</td>
+    <td>
+      <a href="talks/oss-supply-chain---are-prometheu.md">OSS Supply Chain - Are Prometheus exporters the weak link?</a>
+    </td>
+    <td>
+      <a href="speakers/david-de-torres-huerta.md">David de Torres Huerta</a>
+      <br>
+      <b>Sysdig</b>
+    </td>
+  </tr>
+   <tr class="talk">
+    <td>11:45</td>
+    <td>
+      <a href="talks/best-practices-and-pitfalls-of-i.md">Best practices and Pitfalls of Instrumenting Your Cloud-Native Application</a>
+    </td>
+    <td>
+      <a href="speakers/jéssica-lins.md">Jéssica Lins</a>
+      <br>
+      <b>Red Hat</b>
+      <a href="speakers/kemal-akkoyun.md">Kemal Akkoyun</a>
```

Comment:
When you have multiple speakers in one table cell, you'll want to put a `<br>` in between (like in https://github.com/promcon/website/blob/106d2e11b294490a637f1f5f23b177e455eff4a9/content/2019-munich/schedule.md?plain=1#L184-L192).

Reply:
Fixed in new build.